### PR TITLE
[cse release 1.64] - local path provisioner

### PR DESCRIPTION
**Pull request by sprait**
## Description

localPathProvisioner helper
localPathProvisioner localPathProvisioner

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: <kebab-case of a module name> | <1st level dir in the repo>
type: fix | feature | chore
summary: <ONE-LINE of what effectively changes for a user>
impact: <what to expect for users, possibly MULTI-LINE>, required if impact_level is high ↓
impact_level: default | high | low
```

### CODE REVIEW

File: modules/031-local-path-provisioner/images/local-path-provisioner/werf.inc.yaml [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
 shell:
   install:
-    - mkdir /src
-    - git clone --depth 1 --branch v{{$localPathProvisionerVersion}} {{ $.SOURCE_REPO }}/rancher/local-path-provisioner.git /src
     - cd /src
-    - for ifile in $(ls /patches/*.patch); do git apply $ifile; done
-    - go get github.com/prometheus/client_golang@v1.17.0
     - go mod tidy
     - go mod vendor
     - GOPROXY={{ $.GOPROXY }} CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X main.VERSION={{$localPathProvisionerVersion}} -extldflags -static -s -w" -o /local-path-provisioner
```
Comment (RomanenkoDenys) [inline modules/031-local-path-provisioner/images/local-path-provisioner/werf.inc.yaml:42]: GOPROXY needs to be before go mod tidy /go mod vendor

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: modules/031-local-path-provisioner/images/local-path-provisioner/werf.inc.yaml (hunk 1 above)
Comment (sprait) [reply modules/031-local-path-provisioner/images/local-path-provisioner/werf.inc.yaml]: done

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**